### PR TITLE
New API: Get following by resource IDs

### PR DESCRIPTION
**Pull request by FalseChord**
1. New following API: Get following by resource IDs
2. Upload db_schema
3. bug fix : server error when adding a new member without active field

### CODE REVIEW

File: models/follows.go [hunk 1]
```diff
@@ -176,6 +215,7 @@ var (
 
 type FollowingAPIInterface interface {
 	GetFollowing(params map[string]string) (interface{}, error)
+	GetFollowed(resource string, ids []string) (interface{}, error)
 	AddFollowing(params map[string]string) error
 	DeleteFollowing(params map[string]string) error
```
Comment (hcchien) [inline models/follows.go]: can we sort the method?

File: routes/follow.go [hunk 2]
```diff
@@ -107,13 +108,52 @@ func (r *followingHandler) Get(c *gin.Context) {
...
+	}{}
+	c.ShouldBindJSON(&input)
+
+	if len(input.Ids) == 0 {
+		c.JSON(http.StatusBadRequest, gin.H{"Error": "Bad Resource ID"})
+		return
+	}
+
+	switch input.Resource {
+	case "member":
```
Comment (hcchien) [inline routes/follow.go]: should we have indent here?

Comment (FalseChord) [reply routes/follow.go]: The code was formatted by go fmt, it's ok for sure we can change the switch-case indentation, but I guess that following the default style is better for us.

File: models/follows.go (hunk 1 above)
Comment (FalseChord) [reply models/follows.go]: Done :D

**Pull request merged**